lib/containerize_task.rb

```diff
@@ -43,4 +44,17 @@ def write_readme_file(dir)
       File.join(dir, "README.md")
     )
   end
+
+  private
+
+  def copy_workload_setup_files(description, workloads, services, path)
```

Comment:
Assignment Branch Condition size for copy_workload_setup_files is too high. [16.12/15]